testing/trino-product-tests/src/main/java/io/trino/tests/product/kafka/KafkaProductTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.kafka;
+
+import io.trino.tempto.BeforeTestWithContext;
+import io.trino.tempto.ProductTest;
+
+import java.io.IOException;
+import java.util.logging.LogManager;
+
+public abstract class KafkaProductTest
+        extends ProductTest
+{
+    @BeforeTestWithContext
+    public void setup()
+            throws IOException
+    {
+        LogManager.getLogManager().readConfiguration(getClass().getResourceAsStream("/io/trino/tests/product/kafka/logging.properties"));
```

Comment:
this will be executed before all tests. Maybe we could have a static boolean variable to do it only once? See `io.trino.tests.product.TestParquet#initialized`

Reply:
Fixed.